src/color.js

```diff
@@ -0,0 +1,11 @@
+import {styles, supportsColor} from 'chalk'
+
+function color_message (message, color) {
+  if (supportsColor && color in styles) {
+    return styles[color]['open'] + message + styles[color]['close']
```

Comment:
Do we need to `open` and `close` here? What about simply `return styles[color](message)`? 

From what I understand of Chalk the opening and closing is only necessary if you want to mix and match or are working with strings that you do not control.